apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -196,7 +196,7 @@ export default function ListBox({
     if (typeof setCount === 'function' && layout) {
       setCount(layout.qListObject.qSize.qcy);
     }
-  }, [layout]);
+  }, [layout && layout.qListObject.qSize.qcy]);
```

Comment:
this is weird I would expect layout to be a new instance when something on there is updated. Anyway, I would put this as a const above to not have logic in the dep arrray

Reply:
i missed [layout, layout && layout.qListObject.qSize.qcy]

Reply:
@haxxmaxx When we get layout changes back from engine as deltas, I think we "patch" those changes on an existing layout object, that is the explanation to this (I think :)).